languages/csharp/exercises/concept/string-formatting/.docs/introduction.md

```diff
@@ -0,0 +1,68 @@
+There are two principal mechanisms for formatting strings in C#/.NET. Use of `String.Format()` and string interpolation.
+
+## Composite Formatting
+
+`String.Format()` takes a string (referred to in the documentation as a _composite format_) comprising fixed text comprising placeholders (known in the documentation as format items), and a variable number of arguments. The return value resolves each format item using the corresponding argument and combines the resolved values with the fixed text.
```

Comment:
>comprising fixed text comprising placeholders

comprising ... comprising - not sure the intention here but it reads a little odd.

Reply:
replaced the second one with an "and"